bindata/etcd/etcd-common-tools

```diff
@@ -20,3 +20,14 @@ function dl_etcdctl {
   podman rm "${etcdctr}"
   etcdctl version
 }
+
+# execute etcdctl command inside of running etcdctl container
+function exec_etcdctl {
+  local command="$@"
+  local container_id=$(sudo crictl ps --label io.kubernetes.container.name=etcdctl -o json | jq -r '.containers[0].id') || true
```

Comment:
`sudo` here is redundant. We check earlier that the script should be run with EUID as root.